src/App.jsx

```diff
@@ -1,20 +1,56 @@
 import React from "react";
 import axios from 'axios';
+import SignIn from './components/SignIn.jsx';
+import NavMenu from './components/NavMenu.jsx';
+import webMenuIcon from '../dist/resources/menu-24.png';
 
 import RecipeTile from "./components/RecipeTile.jsx";
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-
+      username: '',
+      showProfile: false,
+      showHome: false,
+      showFavorites: false,
+      showNav: false,
+      showLogin: true
```

Comment:
I would think it'd be easier to simply have a single variable that contains a string for each view? So instead of this it would be currentView: "home" or "profile" or "favorites" or "login"

Reply:
Great Idea! I'm making it harder than it needs to be

Reply:
@Heine574  changes have been implemented